loadgen/loadgen.cc

```diff
@@ -1492,7 +1492,25 @@ void StartTest(SystemUnderTest* sut, QuerySampleLibrary* qsl,
     detail("*TestSettings (performance_sample_count_override) can override");
     detail("*Refer to Effective Settings for actual value");
   });
-  loadgen::TestSettingsInternal sanitized_settings(requested_settings, qsl);
+
+  TestSettings test_settings = requested_settings;
+  // Look for Audit Config file to override TestSettings during audit
+  std::ifstream audit_config("audit.config");
+  if (audit_config.good()) {
+    LogDetail([] (AsyncDetail& detail) {
+      detail("Found Audit Config file (audit.config)."
+              " Overriding TestSettings from audit.config file.");
+    });
+    std::string audit_scenario = loadgen::ToString(test_settings.scenario);
+    // Remove Spaces from the string
+    RemoveValue(&audit_scenario, ' ');
+    const std::string generic_model = "*";
+    const std::string audit_config_filename = "audit.config";
```

Comment:
Can you:
1) pull this filename literal up and re-use it above.
2) create a "FileExists" helper function, rather than using the ifstream locally? As it is, the ifstream lingers open for the duration of the test. Putting it into a FileExists helper would close it automatically.